logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/publicuser/search/SearchKnownUsersUseCase.kt

```diff
@@ -26,6 +26,7 @@ import com.wire.kalium.logic.data.publicuser.model.UserSearchResult
 import com.wire.kalium.logic.data.user.UserRepository
 import com.wire.kalium.util.KaliumDispatcher
 import com.wire.kalium.util.KaliumDispatcherImpl
+import io.ktor.util.toLowerCasePreservingASCIIRules
```

Comment:
This is a ktor dependency, maybe we can find a replacement or implement our own to avoid breaking the app when this change to internal (like what happened with concurrent map)

Reply:
goood catch, just autompleting without paying attention, I changed it to the std lib.